### PR TITLE
Us 05

### PR DESCRIPTION
**Pull request by Mark-Spiers**

### CODE REVIEW

File: SQLstructure.sql [hunk 1]
```diff
@@ -47,16 +48,87 @@ VALUES ("Business Development"),
...
-		("Leadership Community", "8"),
-        ("Principal", "7"),
-        ("Manager", "6"),
-        ("Consultant", "5"),
-        ("SeniorAssociate", "4"),
-        ("Associate", "3"),
-        ("Trainee", "2"),
-        ("Apprentice", "1");
+INSERT INTO band(band_name, level, competency)
+VALUES ("Executive", "9",'<div class="modal-body">
```
Comment (Fluffasaurus) [inline SQLstructure.sql]: In the future avoid having html stored in the DB as mentioned this morning

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] All unit tests passing 
+- [ ] All UI tests passing
+- [ ] Code coverage suitable
+- [ ] WAVE toolbar run on new or updated screens
+- [ ] Branch named {feature|hotfix|task}/{SM-.*}
+- [ ] If your pull request depends on any other, please link them
+- [ ] Approval from Tech lead
+- [ ] Approval from team member
+- [ ] Commit messages are meaningful
+- [ ] Target is green https://circleci.com/gh/departmentfortransport/street-manager-frontend/tree/master
```
Comment (matthewdoreilly) [inline .github/pull_request_template.md]: Remove this from the checklist until you get your CI setup.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## Description
+Please explain briefly the changes you made here.
+
+## Checklist
+- [ ] All manual security tests passing (Described here: https://github.com/Broom94/Klattice/wiki/Manual-Security-Testing)
+- [ ] All unit tests passing 
+- [ ] All UI tests passing
+- [ ] Code coverage suitable
+- [ ] WAVE toolbar run on new or updated screens
+- [ ] Branch named {feature|hotfix|task}/{SM-.*}
```
Comment (matthewdoreilly) [inline .github/pull_request_template.md]: This should be `Branch named {feature|hotfix|task}/{US-.*}`

File: README.md [hunk 4]
```diff
@@ -1,12 +1,98 @@
...
+See the list of [contributors](https://github.com/Broom94/Klattice/graphs/contributors) who participated in this project.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details
+
+## Acknowledgments
+
+* Matthew O'Reilly
+* Megan O'Kane
```
Comment (matthewdoreilly) [inline README.md]: Nice README 😀

File: SQLstructure.sql [hunk 5]
```diff
@@ -12,6 +12,7 @@ CREATE TABLE band (
 	band_id INT unsigned auto_increment,
 	band_name varchar(30) not null,
     level enum("9", "8", "7", "6", "5", "4", "3", "2", "1"),
+    competency varchar(6000),
```
Comment (matthewdoreilly) [inline SQLstructure.sql]: 6000?!

File: klattice/src/app/app.component.html [hunk 6]
```diff
@@ -60,7 +89,7 @@ <h4 class="modal-title" id="modal-basic-title">{{ roleName }}</h4>
       <div class="form-group">
         <h5 class="jobSpec">Job Specification</h5>
         <p>{{ roleSummary }}</p>
-        <hr />
+        <br />
```
Comment (matthewdoreilly) [inline klattice/src/app/app.component.html]: Frontend Engineers hate `<br/>`. I'd use CSS (padding or margin) here instead to separate this content.

File: .github/pull_request_template.md (hunk 2 above)
Comment (Mark-Spiers) [reply .github/pull_request_template.md]: will be replaced with travisci link next branch

File: .github/pull_request_template.md (hunk 3 above)
Comment (Mark-Spiers) [reply .github/pull_request_template.md]: done

File: README.md (hunk 4 above)
Comment (Mark-Spiers) [reply README.md]: noice

File: SQLstructure.sql (hunk 5 above)
Comment (Mark-Spiers) [reply SQLstructure.sql]: Some have several hundred words, and the average length of word with space is 6 char. + html currently

File: klattice/src/app/app.component.html (hunk 6 above)
Comment (Mark-Spiers) [reply klattice/src/app/app.component.html]: removed

**Pull request merged**